tests/css/CSSCombBearTest.py

```diff
@@ -0,0 +1,35 @@
+from bears.css.CSSCombBear import CSSCombBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+good_file = """
+a { color: red; text-decoration: underline; }
+"""
+
+
+bad_file = """
+a { color: red; text-decoration: underline }
+"""
+
+"""
+.csscomb.json file should contain:
+    {
+        "exclude": ["node_modules/**"],
+        "verbose": true,
+
+        "always-semicolon": true,
+        "color-case": "lower",
+        "color-shorthand": true,
+        "element-case": "lower",
+        "eof-newline": true,
+        "leading-zero": false,
+        "quotes": "single",
+        "remove-empty-rulesets": true,
+        "strip-spaces": true,
+        "unitless-zero": true,
+        "vendor-prefix-align": true
+    }
+"""
```

Comment:
nah this is coala, the user shouldn't have to write other files than the coafile